src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -210,15 +210,24 @@ private enum Flags : byte
                                               hasInitializer: (_propertyFlags & Flags.HasInitializer) != 0,
                                               isCreatedForFieldKeyword: isCreatedForFieldKeyword,
                                               isEarlyConstructed: isEarlyConstructed);
-                Interlocked.CompareExchange(ref _lazyBackingFieldSymbol, backingField, _lazyBackingFieldSymbolSentinel);
+                if (Interlocked.CompareExchange(ref _lazyBackingFieldSymbol, backingField, _lazyBackingFieldSymbolSentinel) == _lazyBackingFieldSymbolSentinel &&
+                    isCreatedForFieldKeyword)
+                {
+                    // semi auto property should override all accessors.
+                    if ((!HasSetAccessor && !this.IsReadOnly) ||
```

Comment:
>((!HasSetAccessor && !this.IsReadOnly) || [](http://example.com/codeflow?start=23&length=41)

Please add an explicit check for ```IsOverrides```

Reply:
@AlekseyTs Yup I was going to do that. I added the IsOverride assert in another PR just to be sure my assumption is correct :)